docs/readthedocs/source/doc/Friesian/QuickStart/Train_wnd.md

```diff
@@ -0,0 +1,269 @@
+# Feature Engineering for W&D (Wide and Deep Learning) using Friesian
+With Friesian FeatureTable APIs, data scientists and machine learning engineers are able to quickly process datasets of all sizes to generate features for recommender models. This tutorial showcases how to extract features for [Wide and Deep Learning](https://arxiv.org/abs/1606.07792) (W&D) using [movielens dataset](http://files.grouplens.org/datasets/movielens/).
+
+## **1. Key Concepts**
+A **FeatureTable** provides distributed table for processing recsys feature, common tabular operations, as well as high-level recsys feature transformation. e.g.
```

Comment:
This is not correct. FeatureTable is a distributed table for processing RecSys features; it provides both common tabular operations (such as join) and RecSys feature transformations (such as negative sampling).